pkg/notifications/notifications.go

```diff
@@ -1,33 +1,51 @@
 package notifications
```

Comment:
For sharings, I've used a singular for the package in `pkg` and a plural for the package in `web`: `pkg/sharing` and `web/sharings` to avoid the confusion between the two packages when they are used both in a third package. Let's say it's an experiment, but we find it OK, I really hope to apply it to `permissions` (the `pkg/permissions` and `web/permissions` are often used together). Maybe, you can consider it for `notifications` too.